src/openrct2/ride/Vehicle.cpp

```diff
@@ -7419,15 +7412,25 @@ void Vehicle::UpdateAdditionalAnimation()
                     else
                         seat_rotation++;
 
-                    animation_frame = (seat_rotation - 4) & 7;
-                    Invalidate();
+                    int8_t targetSeatRotation = (seat_rotation - 4);
```

Comment:
What's the -4 for?

Reply:
A seat rotation of 4 is 0 degrees:
https://github.com/OpenRCT2/OpenRCT2/blob/1a35cdafb364fe5f94c0edfb4e763371c61b8208/src/openrct2-ui/windows/RideConstruction.cpp#L478

Reply:
@Broxzier